client-misk-dynamodb/src/test/kotlin/app/cash/backfila/client/misk/dynamodb/DynamoDbLastEvaluatedKeyTest.kt

```diff
@@ -44,10 +45,11 @@ class DynamoDbLastEvaluatedKeyTest {
   fun `really long run batches work`() {
     testData.addLinkinPark()
 
-    // TODO(mikepaw) use batch size to change how many items show up in a batch.
+    // Pause so only one dynamo batch runs per runBatch service call
```

Comment:
maybe you want to test by calling the api directly instead? Didn't we have tests like that for hibernate?

Reply:
I'm not quite sure what you are referring to. I want to test that stopwatch I just added in that last PR. I'm going to merge this in and make a follow up PR if I figure out what you are suggesting.